// react-native-localize/mock/jest.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.useLocalize = exports.ServerLanguagesProvider = exports.openAppLanguageSettings = exports.findBestLanguageTag = exports.usesAutoTimeZone = exports.usesAutoDateAndTime = exports.usesMetricSystem = exports.uses24HourClock = exports.getTimeZone = exports.getTemperatureUnit = exports.getNumberFormatSettings = exports.getLocales = exports.getCurrencies = exports.getCountry = exports.getCalendar = void 0;
const mock = require(".");
exports.getCalendar = jest.fn(mock.getCalendar);
exports.getCountry = jest.fn(mock.getCountry);
exports.getCurrencies = jest.fn(mock.getCurrencies);
exports.getLocales = jest.fn(mock.getLocales);
exports.getNumberFormatSettings = jest.fn(mock.getNumberFormatSettings);
exports.getTemperatureUnit = jest.fn(mock.getTemperatureUnit);
exports.getTimeZone = jest.fn(mock.getTimeZone);
exports.uses24HourClock = jest.fn(mock.uses24HourClock);
exports.usesMetricSystem = jest.fn(mock.usesMetricSystem);
exports.usesAutoDateAndTime = jest.fn(mock.usesAutoDateAndTime);
exports.usesAutoTimeZone = jest.fn(mock.usesAutoTimeZone);
exports.findBestLanguageTag = jest.fn(mock.findBestLanguageTag);
exports.openAppLanguageSettings = jest.fn(mock.openAppLanguageSettings);
exports.ServerLanguagesProvider = mock.ServerLanguagesProvider;
exports.useLocalize = jest.fn(mock.useLocalize);
